sigma/types.py

```diff
@@ -689,12 +689,11 @@ class SigmaRegularExpressionFlag(Enum):
     DOTALL = auto()
 
 
-@dataclass
```

Comment:
Would prefer to keep this as dataclass for the provided features, consistency and clean structure.